functions/Set-DbaDbFileGrowth.ps1

```diff
@@ -101,7 +101,7 @@ function Set-DbaDbFileGrowth {
         }
 
         foreach ($db in $InputObject) {
-            $allfiles = @($db.FileGroups.Files, $db.LogFiles)
+            $allfiles = @($db.FileGroups.Files + $db.LogFiles)
```

Comment:
Thanks so much for the PR, Geo! This is failing our tests with the following error
```
Method invocation failed because [Microsoft.SqlServer.Management.Smo.DataFile] does not contain a method named 'op_Addition'.
```

This may be what you're looking for. It's not as pretty but should work

```suggestion
            $allfiles = @($db.FileGroups.Files)
            $allfiles += $db.LogFiles
```

Reply:
I'm doing a release today so I'll go ahead and commit the suggestion and we'll see how it works 👍🏼